Builds/LinuxMakefile/pchbuild.mk

```diff
@@ -325,11 +328,12 @@ $(JUCE_OUTDIR)/test: test.cpp
 	$(V_AT)$(CXX) -o $(JUCE_OUTDIR)/test $(JUCE_OUTDIR)/test.o $(TEST_LDFLAGS) $(RESOURCES) $(TARGET_ARCH)
 
 printlinkcommand:
-	echo link command
-	echo "$(V_AT)$(CXX) -o $(JUCE_OUTDIR)/$(JUCE_TARGET_STANDALONE_PLUGIN) $(OBJECTS_STANDALONE_PLUGIN) $(JUCE_OUTDIR)/$(JUCE_TARGET_SHARED_CODE) $(JUCE_LDFLAGS) $(JUCE_LDFLAGS_STANDALONE_PLUGIN) $(RESOURCES) $(TARGET_ARCH)"
-	echo VST2-depenency
-	echo "$(JUCE_OUTDIR)/$(JUCE_TARGET_VST)"
-	echo VST3-depenency
-	echo "$(JUCE_OUTDIR)/$(JUCE_TARGET_VST3)"
+	@echo "CONFIG=$(CONFIG)"
+	@echo link command
+	@echo "$(V_AT)$(CXX) -o $(JUCE_OUTDIR)/$(JUCE_TARGET_STANDALONE_PLUGIN) $(OBJECTS_STANDALONE_PLUGIN) $(JUCE_OUTDIR)/$(JUCE_TARGET_SHARED_CODE) $(JUCE_LDFLAGS) $(JUCE_LDFLAGS_STANDALONE_PLUGIN) $(RESOURCES) $(TARGET_ARCH)"
+	@echo VST2-depenency
+	@echo "$(JUCE_OUTDIR)/$(JUCE_TARGET_VST)"
+	@echo VST3-depenency
```

Comment:
In Builds/LinuxMakefile/pchbuild.mk lines 334 and 336 have a typo:
`	@echo VST2-depenency `  should be corrected to `	@echo VST2-dependency
`

Reply:
I hope you can forgive me that I overwrote that commit